app/views/users/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here is a specific user</h1>
\ No newline at end of file
+<%= render partial: "shared/user", locals: { user: @user } %>
+<div class="user-bio"> 
+  <h3>Bio</h3>
+  <p><%= @user.bio %>
+</div>
+<div class="user-allposts">
+  <% @user.posts.each do |post| %>
+    <div class="user-post">
+      <h3><%= post.title %></h3>
+      <p><%= post.text %></p>
```

Comment:
- [ ] On the user's show page, it is expected that you show the user's 3 most recent posts but instead you are displaying all the posts.
Please use the `#recent_posts` method which you have already defined in the `User`model 😎 

- [ ] Please create the "See all posts" button which should take us to the user's posts index page as shown in the image below👍 

### Wireframe
![image](https://user-images.githubusercontent.com/46839548/217682737-de2f7c31-7d1d-4e31-8942-a1d835315e3b.png)

### Your design
![image](https://user-images.githubusercontent.com/46839548/217681929-fe0b805b-31e2-4e73-bbbb-28df9042f3b8.png)

Reply:
- method changed to recent_posts
- button created!